src/test/enhanced-debugging.ts

```diff
@@ -0,0 +1,285 @@
+import { sortedUrl, toSortedQueryString } from '../mocks/mock-utils'
+import { Request } from '../request'
+import colors from '@colors/colors'
```

Comment:
This dependency is missing, and the test suite fails without it on CI. For you it might work due to a transitive dependency from Karma.